lib/src/util/dart_type_utilities.dart

```diff
@@ -83,6 +86,9 @@ class DartTypeUtilities {
   static bool isNullLiteral(Expression expression) =>
       expression?.unParenthesized is NullLiteral;
 
+  static bool isUserDefinedOperator(TokenType type) =>
+      _userDefinedOperatorsSet.contains(type);
```

Comment:
Sorry! I should have dug a little further for you. You don't need to define the set yourself, just use `Token.isUserDefinableOperator`.